modules/jdbc-test/src/test/java/org/testcontainers/junit/SimplePgroutingSQLTest.java

```diff
@@ -0,0 +1,32 @@
+package org.testcontainers.junit;
+
+import com.zaxxer.hikari.HikariConfig;
+import com.zaxxer.hikari.HikariDataSource;
+import org.junit.Test;
+import org.testcontainers.containers.JdbcDatabaseContainer;
+import org.testcontainers.containers.PgroutingContainerProvider;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+public class SimplePgroutingSQLTest {
```

Comment:
We don't really need this test, instead a new line in `JDBCDriverTest` would make sense: 

https://github.com/testcontainers/testcontainers-java/blob/f3ae35ef8379bdc248e71768956b73048216e456/modules/jdbc-test/src/test/java/org/testcontainers/jdbc/JDBCDriverTest.java#L56

Reply:
Good point. I didn't notice the JDBCDriverTest.